src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -46,7 +46,9 @@
 
                 <!-- is this a Staff Only filing? -->
                 <div v-if="filing.isTypeStaff" class="item-header__subtitle">
-                  <FiledLabel :filing="filing" />
+                  <span v-if="filing.displayName === 'Correction - Put Back On' ||
+                  filing.displayName === 'Administrative Dissolution'">FILED<FiledLabel :filing="filing" /></span>
```

Comment:
can we use something else to check rather than checking the string? maybe name

Reply:
fixed.